plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -434,21 +435,30 @@ public static LongWriteFunction timeWriteFunction(int precision)
     /**
      * @deprecated This method uses {@link java.sql.Timestamp} and the class cannot represent date-time value when JVM zone had
      * forward offset change (a 'gap'). This includes regular DST changes (e.g. Europe/Warsaw) and one-time policy changes
-     * (Asia/Kathmandu's shift by 15 minutes on January 1, 1986, 00:00:00). If driver only supports {@link LocalDateTime}, use
-     * {@link #timestampColumnMapping} instead.
+     * (Asia/Kathmandu's shift by 15 minutes on January 1, 1986, 00:00:00). This mapping also disables pushdown by default
+     * to ensure correctness because rounding happens within Trino and won't apply to remote system.
+     * If driver only supports {@link LocalDateTime}, use {@link #timestampColumnMapping} instead.
      */
     @Deprecated
-    public static ColumnMapping timestampColumnMappingUsingSqlTimestamp(TimestampType timestampType)
+    public static ColumnMapping timestampColumnMappingUsingSqlTimestampWithRounding(TimestampType timestampType)
```

Comment:
I'm not a fan of this name. I think we can leave it unchanged. If we really want a name with all that information in it, I think `roundingSqlTimestampMapping` or `roundingSqlTimestampColumnMapping` would be better.